source/administration/changelog.md

```diff
@@ -6,7 +6,11 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.24 - [Feature Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
 
-**Release day: 2020-06-16**
+- **v5.24.1, release day TBD**
+  - Fixing an issue with a SemVer violation that breaks all plugins that use the ``plugin-api``. [MM-26231](https://mattermost.atlassian.net/browse/MM-26231)
+  - Fixing an issue with the Plugin Tooltip implementation that causes some links to be truncated when rendered. [MM-25808](https://mattermost.atlassian.net/browse/MM-25808)
```

Comment:
@mickmister Please help with the following details for communicating the dot release:

1. A better description if needed.
2. How the bug affects customers and why they should upgrade.
3. What is the workaround until this is fixed.